src/components/tasks/card/card.module.scss

```diff
@@ -101,3 +118,111 @@ $black: #252525;
   text-align: left;
   cursor: pointer;
 }
+
+.taskTagLevelWrapper{
+  display: flex;
+  padding-top: .5rem;
```

Comment:
Would it be better to write `0.5` than `.5`?

Reply:
This applies for other places in this file as well

Reply:
yes done